core/src/main/java/de/bwaldvogel/mongo/backend/AbstractMongoBackend.java

```diff
@@ -336,4 +345,21 @@ public void setClock(Clock clock) {
         this.clock = clock;
     }
 
+    @Override
+    public void disableOplog() {
+        oplog = NoopOplog.get();
+    }
+
+    @Override
+    public void enableOplog() {
+        MongoDatabase localDatabase = resolveDatabase("local");
+        MongoCollection<Document> collection;
+        try {
+            collection = (MongoCollection<Document>) localDatabase.createCollectionOrThrowIfExists(OPLOG_COLLECTION_NAME, CollectionOptions.withDefaults());
```

Comment:
should we just check for existence of the collection rather than relying on exception handling?